packages/accessibility/src/AccessibilityManager.js

```diff
@@ -270,6 +278,15 @@ export class AccessibilityManager
      */
     update()
     {
+        /* On Android, tab order seems to be calculated by position rather than tabIndex,
```

Comment:
When you say Android, is this all browsers on Android? Or Chrome? Firefox? With hacky things like this, it'd be nice to know a more precise scenario to know why this exists.

Reply:
As far as I know, this is the standard web browser for all android devices post (4.4) that we tested on. Will amend to reflect that!